### PR TITLE
Enable tablet mode for phones

### PR DESCRIPTION
**Pull request by Ahmed9914**
Issue #806

### CODE REVIEW

File: app/src/main/res/values/booleans.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
-  <bool name="is_tablet">false</bool>
+  <bool name="is_tablet">true</bool>
```
Comment (ozbek) [inline app/src/main/res/values/booleans.xml]: Rather than setting this to true, I'd remove it, and all references and methods using it.

Comment (Ahmed9914) [inline app/src/main/res/values/booleans.xml]: I'll work on that.

Comment (selmanon) [inline app/src/main/res/values/booleans.xml]: @Ahmed9914 Do you mean that you will work on issue #806 ?

**Comment by ahmedre** [comment]
jazakAllah khairan for doing this - i'll merge this, but can you make another patch to rename "isTablet" to something like "isDualPagesMode" or something?

**Comment by Ahmed9914** [comment]
I think you mean to change `prefs_tablet_mode_title` from "Tablet Mode" to "Dual page mode" as the method `isTablet` is checking the screen width, so its name is suitable to its functionality.

Please clearify this point.

JazakAllah khairan

**Comment by ahmedre** [comment]
i mean both - isTablet is only used for dual screen mode, so the variable name shouldn't contain anything about tablets in it anymore (neither should the strings).

**Comment by ozbek** [comment]
@Ahmed9914, while you are at it, would you please also remove `bool name="is_tablet"` definitions from other `values-xxx` folders? Also, please note that there is already `QuranUtils.isDualPages()` method.

File: app/src/main/res/values/booleans.xml (hunk 1 above)
Comment (ozbek) [inline app/src/main/res/values/booleans.xml]: @selmanon, apparently, he meant to work on my above comment :)

**Comment by Ahmed9914** [comment]
@selmanon sorry, I didn't notice your question. yes as @ozbek said , I meant to work on my above comment

**Pull request merged**